.eslintrc.js

```diff
@@ -1,21 +1,39 @@
 module.exports = {
```

Comment:
If we are going to update this file what do you think about conforming it to the [same in the module template](https://github.com/MetaMask/metamask-module-template/blob/c4fda7b5fab49fa84cf0ce7f4fe3981eca2fe23f/.eslintrc.js)? It seems like it's pretty close already.

Reply:
I tried this and I got 7 of the following errors, what is the preferred solution to these?

`error  Unexpected use of 'Buffer'. This global is not available in the Node.js and browser environment  no-restricted-globals`

Reply:
@mcmire @Gudahtt `@metamask/eslint-config-browser` is not working for me, but maybe I just don't understand the details of extends and overrides and the order

I can of course put `/* eslint-disable no-restricted-globals */` at the top of the file

Reply:
@HowardBraham Yeah, so it looks like this library has the same sort of caveat that `eth-ledger-bridge-keyring` has: it implicitly relies on Buffer to be present (meaning that if someone wants to use this in a browser context, they need to supply a polyfill for Buffer or use the `buffer` package). Probably best to point this requirement out in the README and then exclude Buffer from being restricted here, by adding a new rule in `rules`:

``` js
const environmentRules = require('@metamask/eslint-config/src/environment.json');

module.exports = {
  // ...

  rules: {
    // ...
    'no-restricted-globals': [
      'error',
      ...environmentRules['no-restricted-globals'].filter(
        (rule) => typeof rule !== 'string' && !['Buffer'].includes(rule.name),
      ),
    ],
  },
  
  // ...
};
```